### ISSUE TITLE
Error fetch database when CSV added to doltlab instance via command-line

### ISSUE THREAD

**Issue by Analect** [dolthub/doltlab-issues#39]
Hi. Thanks for your work on Doltlab, which I've been experimenting with.

I hit an issue when testing the import of a relatively small CSV file (307 rows, 12 columns) to doltlab. I'm on version 0.7 (having upgraded from 0.6 ... ie. docker volumes date back to v. 0.6).

I ran:
```
dolt table import -c --pk=id test_table ~/dev/rstudio/test_table.csv
```
... and got acknowledgement that the import worked.  However, back on the doltlab UI, I was getting this "error fetching database" 

![image](https://user-images.githubusercontent.com/4063815/201470684-04169761-0b32-452b-9231-c7b8f36d7be0.png)

If I access the database from the command-line, it seems it has imported OK.  I tried to run a revert on the commit (`select dolt_revert('<commit-id>`) ... which succeeded ... to see if that might fix things, but to no avail.

```
STATUS                 PORTS
                                                                           NAMES
af95adf0833b   public.ecr.aws/dolthub/doltlab/dolthub-server:v0.7.0              "docker-entrypoint.s…"   2 weeks ago   Up 36 minutes          3000/tcp
                                                                           doltlab_doltlabui_1
96838ffea2ce   public.ecr.aws/dolthub/doltlab/dolthubapi-graphql-server:v0.7.0   "docker-entrypoint.s…"   2 weeks ago   Up 36 minutes          9000/tcp
                                                                           doltlab_doltlabgraphql_1
3eb3b04dc08c   public.ecr.aws/dolthub/doltlab/dolthubapi-server:v0.7.0           "/app/go/services/do…"   2 weeks ago   Up 36 minutes
                                                                           doltlab_doltlabapi_1
c2ae0f01be23   public.ecr.aws/dolthub/doltlab/doltremoteapi-server:v0.7.0        "/app/go/services/do…"   2 weeks ago   Up 36 minutes          0.0.0.0:50051->50051/tcp, :::50051->50051/tcp
                                                                           doltlab_doltlabremoteapi_1
e7abe2ace69c   public.ecr.aws/dolthub/doltlab/fileserviceapi-server:v0.7.0       "/app/go/services/fi…"   2 weeks ago   Up 36 minutes
                                                                           doltlab_doltlabfileserviceapi_1
dc748eb44b0e   envoyproxy/envoy-alpine:v1.18-latest                              "/docker-entrypoint.…"   2 weeks ago   Up 36 minutes          0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:100->100/tcp, :::100->100/tcp, 0.0.0.0:4321->4321/tcp, :::4321->4321/tcp, 0.0.0.0:7770->7770/tcp, :::7770->7770/tcp, 10000/tcp   doltlab_doltlabenvoy_1
a0f592db2f1a   public.ecr.aws/dolthub/doltlab/postgres-server:v0.7.0             "docker-entrypoint.s…"   2 weeks ago   Up 36 minutes          5432/tcp
                                                                           doltlab_doltlabdb_1
```
I notice on inspecting `doltlab_doltlabfileserviceapi_1` and see it complaining about not being able to access `doltlabenvoy:9480`.  On inspecting the docker-compose in 0.7 it seems there is no port 9480 being exposed.  `~/dev/doltlab$ cat docker-compose.yaml | grep 9480`.  Could that be the source of my problem?

```
docker logs doltlab_doltlabfileserviceapi_1
2022-11-12T10:27:50.887Z        INFO    server/main.go:132      server is shutting down
2022-11-12T10:27:51.198Z        INFO    server/main.go:151      Server stopped.
2022-11-12T10:28:04.539Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Channel created      {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.540Z        INFO    zap/grpclogger.go:73    [core][Channel #3] original dial target is: "doltlabenvoy:9480" {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.540Z        INFO    zap/grpclogger.go:73    [core][Channel #3] parsed dial target is: {Scheme:doltlabenvoy Authority: Endpoint:9480 URL:{Scheme:doltlabenvoy Opaque:9480 User: Host: Path: RawPath: OmitHost:false ForceQuery:false RawQuery: Fragment: RawFragment:}}      {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.540Z        INFO    zap/grpclogger.go:73    [core][Channel #3] fallback to scheme "passthrough"    {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.540Z        INFO    zap/grpclogger.go:73    [core][Channel #3] parsed dial target is: {Scheme:passthrough Authority: Endpoint:doltlabenvoy:9480 URL:{Scheme:passthrough Opaque: User: Host: Path:/doltlabenvoy:9480 RawPath: OmitHost:false ForceQuery:false RawQuery: Fragment: RawFragment:}}     {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.541Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Channel authority set to "dolthubapi{"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.541Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Resolver state updated: {
  "Addresses": [
    {
      "Addr": "doltlabenvoy:9480",
      "ServerName": "",
      "Attributes": null,
      "BalancerAttributes": null,
      "Type": 0,
      "Metadata": null
    }
  ],
  "ServiceConfig": null,
  "Attributes": null
} (resolver returned new addresses)     {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.541Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Channel switches to new LB policy "pick_first"       {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.541Z        INFO    zap/grpclogger.go:73    [core][Channel #3 SubChannel #4] Subchannel created    {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.546Z        INFO    zap/grpclogger.go:73    [core][Channel #1 SubChannel #2] Subchannel Connectivity change to CONNECTING   {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.546Z        INFO    zap/grpclogger.go:73    [core][Channel #1 SubChannel #2] Subchannel picks a new address "doltlabenvoy:9480" to connect  {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.562Z        INFO    zap/grpclogger.go:73    [core][Channel #3 SubChannel #4] Subchannel Connectivity change to CONNECTING   {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.562Z        INFO    zap/grpclogger.go:73    [core][Channel #3 SubChannel #4] Subchannel picks a new address "doltlabenvoy:9480" to connect  {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.562Z        INFO    zap/grpclogger.go:73    [core]pickfirstBalancer: UpdateSubConnState: 0xc0003fa670, {CONNECTING <nil>}   {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.562Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Channel Connectivity change to CONNECTING    {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.562Z        INFO    server/main.go:144      Serving fileserviceapi server.  {"addr": ":60055"}
2022-11-12T10:28:04.562Z        INFO    zap/grpclogger.go:73    [core]pickfirstBalancer: UpdateSubConnState: 0xc0003ec390, {CONNECTING <nil>}   {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:04.562Z        INFO    zap/grpclogger.go:73    [core][Channel #1] Channel Connectivity change to CONNECTING    {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:06.593Z        WARN    zap/grpclogger.go:85    [core][Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
  "Addr": "doltlabenvoy:9480",
  "ServerName": "eventsapi",
  "Attributes": null,
  "BalancerAttributes": null,
  "Type": 0,
  "Metadata": null
}. Err: connection error: desc = "transport: Error while dialing dial tcp 172.19.0.3:9480: connect: connection refused"{"system": "grpc", "grpc_log": true}
github.com/grpc-ecosystem/go-grpc-middleware/logging/zap.(*zapGrpcLoggerV2).Warningln
        external/com_github_grpc_ecosystem_go_grpc_middleware/logging/zap/grpclogger.go:85
google.golang.org/grpc/internal/grpclog.WarningDepth
        external/org_golang_google_grpc/internal/grpclog/grpclog.go:46
google.golang.org/grpc/grpclog.(*componentData).WarningDepth
        external/org_golang_google_grpc/grpclog/component.go:41
google.golang.org/grpc/internal/channelz.AddTraceEvent
        external/org_golang_google_grpc/internal/channelz/funcs.go:342
google.golang.org/grpc/internal/channelz.Warningf
        external/org_golang_google_grpc/internal/channelz/logging.go:59
google.golang.org/grpc.(*addrConn).createTransport
        external/org_golang_google_grpc/clientconn.go:1278
google.golang.org/grpc.(*addrConn).tryAllAddrs
        external/org_golang_google_grpc/clientconn.go:1213
google.golang.org/grpc.(*addrConn).resetTransport
        external/org_golang_google_grpc/clientconn.go:1148
google.golang.org/grpc.(*addrConn).connect
        external/org_golang_google_grpc/clientconn.go:803
2022-11-12T10:28:06.594Z        INFO    zap/grpclogger.go:73    [core][Channel #1 SubChannel #2] Subchannel Connectivity change to TRANSIENT_FAILURE    {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:06.595Z        INFO    zap/grpclogger.go:73    [core]pickfirstBalancer: UpdateSubConnState: 0xc0003ec390, {TRANSIENT_FAILURE connection error: desc = "transport: Error while dialing dial tcp 172.19.0.3:9480: connect: connection refused"}  {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:06.595Z        INFO    zap/grpclogger.go:73    [core][Channel #1] Channel Connectivity change to TRANSIENT_FAILURE     {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:06.595Z        WARN    zap/grpclogger.go:85    [core][Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
  "Addr": "doltlabenvoy:9480",
  "ServerName": "dolthubapi",
  "Attributes": null,
  "BalancerAttributes": null,
  "Type": 0,
  "Metadata": null
}. Err: connection error: desc = "transport: Error while dialing dial tcp 172.19.0.3:9480: connect: connection refused"{"system": "grpc", "grpc_log": true}
github.com/grpc-ecosystem/go-grpc-middleware/logging/zap.(*zapGrpcLoggerV2).Warningln
        external/com_github_grpc_ecosystem_go_grpc_middleware/logging/zap/grpclogger.go:85
google.golang.org/grpc/internal/grpclog.WarningDepth
        external/org_golang_google_grpc/internal/grpclog/grpclog.go:46
google.golang.org/grpc/grpclog.(*componentData).WarningDepth
        external/org_golang_google_grpc/grpclog/component.go:41
google.golang.org/grpc/internal/channelz.AddTraceEvent
        external/org_golang_google_grpc/internal/channelz/funcs.go:342
google.golang.org/grpc/internal/channelz.Warningf
        external/org_golang_google_grpc/internal/channelz/logging.go:59
google.golang.org/grpc.(*addrConn).createTransport
        external/org_golang_google_grpc/clientconn.go:1278
google.golang.org/grpc.(*addrConn).tryAllAddrs
        external/org_golang_google_grpc/clientconn.go:1213
google.golang.org/grpc.(*addrConn).resetTransport
        external/org_golang_google_grpc/clientconn.go:1148
google.golang.org/grpc.(*addrConn).connect
        external/org_golang_google_grpc/clientconn.go:803
2022-11-12T10:28:06.595Z        INFO    zap/grpclogger.go:73    [core][Channel #3 SubChannel #4] Subchannel Connectivity change to TRANSIENT_FAILURE    {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:06.596Z        INFO    zap/grpclogger.go:73    [core]pickfirstBalancer: UpdateSubConnState: 0xc0003fa670, {TRANSIENT_FAILURE connection error: desc = "transport: Error while dialing dial tcp 172.19.0.3:9480: connect: connection refused"}  {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:06.596Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Channel Connectivity change to TRANSIENT_FAILURE     {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:07.595Z        INFO    zap/grpclogger.go:73    [core][Channel #1 SubChannel #2] Subchannel Connectivity change to IDLE {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:07.595Z        INFO    zap/grpclogger.go:73    [core]pickfirstBalancer: UpdateSubConnState: 0xc0003ec390, {IDLE connection error: desc = "transport: Error while dialing dial tcp 172.19.0.3:9480: connect: connection refused"}       {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:07.595Z        INFO    zap/grpclogger.go:73    [core][Channel #1] Channel Connectivity change to IDLE {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:07.597Z        INFO    zap/grpclogger.go:73    [core][Channel #3 SubChannel #4] Subchannel Connectivity change to IDLE {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:07.597Z        INFO    zap/grpclogger.go:73    [core]pickfirstBalancer: UpdateSubConnState: 0xc0003fa670, {IDLE connection error: desc = "transport: Error while dialing dial tcp 172.19.0.3:9480: connect: connection refused"}       {"system": "grpc", "grpc_log": true}
2022-11-12T10:28:07.597Z        INFO    zap/grpclogger.go:73    [core][Channel #3] Channel Connectivity change to IDLE {"system": "grpc", "grpc_log": true}
```

**Comment by coffeegoddd** [comment]
Hey @Analect, thanks for the issue!

So the `doltlabfileserviceapi` _should_ be able to connect to `doltlabenvoy:9480`, but the port isn't listed in the `docker-compose.yaml`, it's in the `envoy.yaml` as `outbound_grpc_listener`. The services also use this port for outbound communication by default. 

It's strange that you're seeing these connection failures. You can try an attempt to reach that endpoint with:
```bash
docker run -it --rm --network doltlab curlimages/curl:latest http://doltlabenvoy:9480
```
Then in the logs of `doltlabenvoy`, you should see something like:
```bash
[2022-11-12T14:05:30.647Z] "GET / HTTP/1.1" 404 Unimplemented NR 0 0 0 - "xxx.xxx.xxx.x" "curl/7.86.0-DEV" "02817954-6781-448e-b07c-4c1620f579c2" "doltlabenvoy:9480" "-" "-"
```
which would indicate the port is reachable inside the `doltlab` network, meaning `doltlabfileserviceapi` should be able to reach it as well.

Can you also try stopping DoltLab, removing all of the containers and the docker network (do not remove the docker volumes), then restarting it with the `./start-doltlab.sh` script? If there are non-precious docker images/containers on the doltlab host you can do this with `docker system prune -a`.

I'm gonna follow up with you in Discord with some requests as well.

**Comment by Analect** [comment]
@coffeegoddd ... thanks. I was able to stop, rm and prune v0.7.0 setup and upgrade to v0.7.1 and the problems described above appear to have been resolved.  The only thing I noticed when pruning was that two docker networks existed (see below).  When I previously upgraded from v0.6 of DoltLab to v0.7, while I stopped and removed the containers of the old, I didn't prune images/networks. It could have been that the `./start-doltlab.sh` might have created a new network `doltlab_doltlab`, if `doltlab` pre-existed ... eventhough I see in your docs that it shouldn't try to recreate a new one if one already exists.  That's the only explanation I have for the two networks being there.
```
doltlab
doltlab_doltlab
```
